src/highrl/envs/eval_env.py

```diff
@@ -90,7 +90,7 @@ def _generate_obstacles_points(
                 new_height = randint(min_dim, max_dim)
                 new_obstacle = SingleObstacle(px, py, new_width, new_height)
                 overlap = self.robot.is_overlapped(
-                    new_obstacle, check_target="robot"
+                    new_obstacle, check_target="agent"
```

Comment:
why did we change robot to agent  ?

Reply:
More generic + it is in the agent.py file so makes more sense.